### ISSUE TITLE
Some pics not visible

### ISSUE THREAD

**Issue by Elisagig** [Mergpijp/GlobalOrthodoxy#248]
A number of colleagues noted this problem

It seems a number of pictures dded aren’t showing up. Might just be maintenance or even something on my end but I know I’ve added many of these pictures. I’m in the right place, yes?https://fourcornersoftheworld.ptrs.ru.nl/researcher/

**Comment by Woseseltops** [comment]
Hey @Elisagig , sorry for the late reply... do you have an example of a picture not showing up? This one perhaps? https://fourcornersoftheworld.ptrs.ru.nl/publication/1158/detail_view/

**Comment by Elisagig** [comment]
Yes this is what we mean. The file shows up but if you click on it you get permission error. The strange thing is that some colleagues are saying sometimes photos show and sometimes they don't. Other colleagues are saying they don't show up. It seems a bit weird.
